### PR TITLE
[Accounts] Update bindings for Xcode 13.0 beta 1

### PR DESCRIPTION
**Pull request by rachelkang**
WIP - need to figure out why Catalyst tests are failing

### CODE REVIEW

**Comment by tj-devel709** [comment]
Does this framework have a MacCatalyst todo file?

**Comment by rachelkang** [comment]
> Does this framework have a MacCatalyst todo file?

@tj-devel709 nope! Based on what I saw on web docs and my chat with @rolfbjarne, I thought I had to add [NoMacCatalyst] to these APIs as well, but I removed them because that caused xtro errors

**Review by rolfbjarne — APPROVED** [review]

**Review by mandel-macaque — APPROVED** [review]

**Review by tj-devel709 — APPROVED** [review]

**Comment by mandel-macaque** [comment]
Test failure is unrelated. I updated it again main since it is using Xcode13 beta2, everything should pass.

File: src/accounts.cs [hunk 1]
```diff
@@ -61,6 +69,10 @@ interface ACAccountCredential : NSSecureCoding {
 	delegate void ACAccountStoreRemoveCompletionHandler (bool success, NSError error);
 	delegate void ACRequestCompletionHandler (bool granted, NSError error);
 	
+	[Introduced (PlatformName.iOS, 6, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
+	[Deprecated (PlatformName.iOS, 15, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
+	[Introduced (PlatformName.MacOSX, 10, 8, message: "Use the non-Apple SDK relating to your account type instead.")]
+	[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
```
Comment (dalexsoto) [inline src/accounts.cs:75]: Same

```suggestion
	[Deprecated (PlatformName.iOS, 15, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
	[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
```

File: src/accounts.cs [hunk 2]
```diff
@@ -111,6 +123,10 @@ interface ACAccountStore {
 		void RemoveAccount (ACAccount account, ACAccountStoreRemoveCompletionHandler completionHandler);
 	}
 
+	[Introduced (PlatformName.iOS, 6, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
+	[Deprecated (PlatformName.iOS, 15, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
+	[Introduced (PlatformName.MacOSX, 10, 8, message: "Use the non-Apple SDK relating to your account type instead.")]
+	[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
```
Comment (dalexsoto) [inline src/accounts.cs:129]:
```suggestion
	[Deprecated (PlatformName.iOS, 15, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
	[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
```

File: src/social.cs [hunk 3]
```diff
@@ -60,6 +60,10 @@ interface SLRequest {
 		[Export ("requestForServiceType:requestMethod:URL:parameters:")]
 		SLRequest Create (NSString serviceType, SLRequestMethod requestMethod, NSUrl url, [NullAllowed] NSDictionary parameters);
 
+		[Introduced (PlatformName.iOS, 6, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
+		[Deprecated (PlatformName.iOS, 15, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
+		[Introduced (PlatformName.MacOSX, 10, 8, message: "Use the non-Apple SDK relating to your account type instead.")]
+		[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
```
Comment (dalexsoto) [inline src/social.cs:66]: Same

```suggestion
		[Deprecated (PlatformName.iOS, 15, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
		[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
```

File: src/twitter.cs [hunk 4]
```diff
@@ -19,6 +19,11 @@ namespace Twitter {
 	[Availability (Deprecated = Platform.iOS_6_0, Message = "Use the 'Social' framework.")]
 	[BaseType (typeof (NSObject))]
 	interface TWRequest {
+		
+		[Introduced (PlatformName.iOS, 6, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
+		[Deprecated (PlatformName.iOS, 15, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
+		[Introduced (PlatformName.MacOSX, 10, 8, message: "Use the non-Apple SDK relating to your account type instead.")]
+		[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use the non-Apple SDK relating to your account type instead.")]
```
Comment (dalexsoto) [inline src/twitter.cs:26]: Not needed, the parent class is already deprecated.

```suggestion
```

**Review by dalexsoto — CHANGES_REQUESTED** [review]
Some comments here :)

**Comment by rachelkang** [comment]
intro and xtro are all passing!

**Comment by rachelkang** [comment]
not sure why build is failing

**Pull request merged**